oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,213 @@
+encourage a commitment of 20 hours/month from each of an organization's Core Contributors, however, we
+recognize that circumstances differ and a different commitment may be more appropriate. Before signing
+the agreement, please reach out to the Program Administrators to discuss.
+
+Questions about the Program can be directed to the tCRIL administrator(s) via the ``#core-contributors`` Slack room
+in the `Open edX Slack <https://openedx.slack.com/>`_.
+
+.. _selecting new members:
+
+Adding New Core Contributors
+----------------------------
+
+New Core Contributors are determined via a nomination process - CCs may nominate
+new members, or existing community members may put themselves forward for a role.
+
+Current project CCs should always be on the lookout for contributors who demonstrate the
+"3 Cs" (Commitment, Conduct, and Caliber), follow the `Code of Conduct
+<https://open.edx.org/code-of-conduct/>`_, are respectful of other community members, are polite and
+welcoming to all in the community, and have been an overall active community member for some time.
+When a person like this comes to the attention of the CC, the CC should reach out and ask if
+that person would like to be nominated as a CC.
+
+Community members who feel they fit these criteria listed above should feel empowered to
+nominate themselves, as well.
+
+Note that when a new Core Contributor role is being defined and piloted, however,
+nominations should proceed only once the Program Administrator is confident that new Core
+Contributors in that role can be supported. Reach out to the Administrator when there is
+any uncertainty around the status of the role; Administrators can be contacted at ``cc-program-admins@tcril.org``.
+
+New Core Contributor Nomination Process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The process for nomination is as follows:
+
+#. An existing CC ("sponsor") choses to nominate a community member ("sponsored candidate") for a new CC
+   position, OR, a current community member ("organic candidate") decides to put themselves forth for nomination.
+
+#. The sponsor or organic candidate posts a new public nomination thread on Discourse, in the `Core Contributor
+   discussion space <https://discuss.openedx.org/c/working-groups/core-contributors/36>`_.
+
+   #. This post introduces the candidate, describes in a few paragraphs why they are suited to join the program,
+      and provides links to examples of previous Open edX work (for example, pull requests).
+   
+   #. The post should also indicate the candidate's expected commitment to contribution - if the candidate is
+      part of an organization, this should be covered as part of the organization's signature to the Declaration
+      of Commitment to the Core Contributor Program, as covered in the `program administration`_ section.
+   
+   #. Finally, the post must mention the date when the comment period ends, which is one month from the start of
```

Comment:
Does this mean there's effectively a one month lag on someone becoming a CC?  Maybe that's fine; I can imagine situations where a project would really benefit from someone completing their nomination on a speedier timeline, though.  (I'm not proposing a change here, just mulling that over)

Reply:
Going to resolve this as we discuss in another thread